pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/ddp/tissue/TissueSMIDDao.java

```diff
@@ -110,6 +116,22 @@ public String createNewSMIDForTissueWithValue(String tissueId, String userId, St
         }
     }
 
+    /**
+     * Deletes the tissue and smid
+     */
+    public void deleteTissueAndSmId(int tissueId) {
+        SimpleResult results = inTransaction((conn) -> {
+            SimpleResult dbVals = new SimpleResult();
+            DaoUtil.deleteSingleRowById(tissueId, DELETE_SM_ID);
```

Comment:
I could be mistaken about the usage here, but because of 1:n relationship between tissue and sm_id, this can delete more than one row which will throw an exception in the `DaoUtil.deleteSingleRowById`

Reply:
Good catch!  I've changed this to delete the smid by its pk instead of tissue id.